api/endpoints/student.py

```diff
@@ -25,7 +25,7 @@ def on_get(self, request, response):
 		students = self.search(search_string)
 		if students:
 			for student in students:
-				response.media.append(student.dict(exclude=["student_id"]))
+				response.media.append(student.dict())
```

Comment:
This needs to be changed back.  We don't want to send student IDs back to the user